utils/utils.c

```diff
@@ -66,8 +66,18 @@ initGbConfig(void)
 
   gbConf->glfsLruCount = LRU_COUNT_DEF;
   gbConf->logLevel = GB_LOG_INFO;
-  snprintf(gbConf->logDir, PATH_MAX, "%s", GB_LOGDIR_DEF);
   gbConf->cliTimeout = CLI_TIMEOUT_DEF;
+  snprintf(gbConf->logDir, PATH_MAX, "%s", GB_LOGDIR_DEF);
+  snprintf(gbConf->daemonLogFile, PATH_MAX, "%s/gluster-blockd.log",
+           gbConf->logDir);
+  snprintf(gbConf->cliLogFile, PATH_MAX, "%s/gluster-block-cli.log",
+           gbConf->logDir);
+  snprintf(gbConf->gfapiLogFile, PATH_MAX, "%s/gluster-block-gfapi.log",
+           gbConf->logDir);
+  snprintf(gbConf->configShellLogFile, PATH_MAX,
+           "%s/gluster-block-configshell.log", gbConf->logDir);
+  snprintf(gbConf->cmdhistoryLogFile, PATH_MAX, "%s/cmd_history.log",
+           gbConf->logDir);
```

Comment:
Could you also create these log files (empty) when doing `make install`? That way it is easy to find out that they are not packaged/listed in the `gluster-block.spec`.

In the `gluster-block.spec` those files can be marked as `%ghost` so that they are not included in the archive, but will belong to the RPM once they are created.

Reply:
@nixpanic 
Yeah, this is very cool. Will add this.
Thanks.

Reply:
@nixpanic 
BTW, will the following runtime files make sense here about this ?

/run/gluster-blockd.lock
/run/gluster-blockd.socket

Thanks.

Reply:
Files under `/run` are normally created through a `/etc/tmpfiles.d` dropin file. Check `man tmpfiles.d` for details. The files under `/run` should be listed as `%ghost` in the .spec too.

Reply:
Cool, will go through this and as discussed will raise one new PR to fix it.
Thanks.